### PR TITLE
zcash_client_sqlite: Fix tests that broke with the addition of Orchard UA receivers.

### PR DESCRIPTION
**Pull request by nuttycom**
These tests were broken by #1291

### CODE REVIEW

**Review by str4d — DISMISSED** [review]
utACK 72c346c8ddeabba1f3fdd212fc35863d637b23d2

File: zcash_client_sqlite/src/wallet/init.rs [hunk 1]
```diff
@@ -1482,10 +1484,13 @@ mod tests {
                 assert_eq!(DiversifierIndex::from(tv.diversifier_index), di);
                 assert_eq!(tvua.transparent(), ua.transparent());
                 assert_eq!(tvua.sapling(), ua.sapling());
+                #[cfg(not(feature = "orchard"))]
                 assert_eq!(tv.unified_addr, ua.encode(&Network::MainNetwork));
 
+                // hardcoded with knowledge of what's coming next
+                let ua_request = UnifiedAddressRequest::unsafe_new(false, true, true);
```
Comment (str4d) [inline zcash_client_sqlite/src/wallet/init.rs:1491]: Should this also be `UA_TRANSPARENT` like before?

**Review by str4d — APPROVED** [review]
Re-utACK 6f1e4aff1acceedd55244857901678ae1fc900a7 with question.

File: zcash_client_sqlite/src/wallet/init.rs (hunk 1 above)
Comment (nuttycom) [reply zcash_client_sqlite/src/wallet/init.rs:1491]: No; the whole test is under the `transparent-inputs` feature.

**Review by daira — COMMENTED** [review]
Post-hoc ACK 6f1e4aff1acceedd55244857901678ae1fc900a7

**Pull request merged**